test/unit/setup-wp-aliases.js

```diff
@@ -6,11 +6,9 @@ global.wp = {
 	},
 };
 
-[
-	'element',
-	'blocks',
-].forEach( ( entryPointName ) => {
-	Object.defineProperty( global.wp, entryPointName, {
-		get: () => require( entryPointName ),
-	} );
+Object.defineProperty( global.wp, 'element', {
+	get: () => require( 'packages/element' ),
```

Comment:
How is it that `packages/element` refers to the `packages/element` folder, when it's not being defined as a path (and even if it were, required from `test/unit`)?

Reply:
Jest resolves those paths starting from the root dir, that's why it works. 

Our goal is to get rid of this workaround completely when we have all modules moved to packages and we enforce using `createElement` in files using JSX - related commit https://github.com/WordPress/gutenberg/pull/6828/commits/69af0a483ea57f4831984b2c72dad9d4a56c904d from #6828.

The same applies to `wb.blocks` which is here temporarily until we remove deprecations which use this global explicitly.